src/vivarium/framework/results/manager.py

```diff
@@ -108,13 +122,29 @@ def on_post_setup(self, _: Event):
                     else:
                         # We are aggregating the entire population so create a single-row index
                         idx = pd.Index(["all"], name="stratification")
+
                     # Initialize a zeros dataframe
                     self._metrics[measure] = pd.DataFrame(
                         data=0.0,
                         columns=["value"],
                         index=idx,
                     )
 
+        if unused_stratifications:
+            self.logger.info(
+                "The following Stratifications are registered but not used by any "
+                f"Observers: \n{sorted(list(unused_stratifications))}"
+            )
+        if all_missing_stratifications:
```

Comment:
Nit: I think I prefer `all_missing_stratifications` to be `missing_stratifications` and what you currently call `missing_stratifications` to be something like `observation_missing_stratifications` because this line `if all_missing_stratifications:` makes me think it means you're checking if all stratifications are missing.

Reply:
I was really struggling w/ naming these things here!